backend/benefit/applications/tests/test_ahjo_integration.py

```diff
@@ -737,11 +742,17 @@ def test_get_applications_for_open_case(
             ahjo_status.save()
 
     applications_for_open_case = Application.objects.get_by_statuses(
-        [ApplicationStatus.HANDLING], AhjoStatusEnum.SUBMITTED_BUT_NOT_SENT_TO_AHJO
+        [
+            ApplicationStatus.HANDLING,
+            ApplicationStatus.ACCEPTED,
+            ApplicationStatus.REJECTED,
+        ],
+        AhjoStatusEnum.SUBMITTED_BUT_NOT_SENT_TO_AHJO,
+        True,
     )
     # only handled_applications should be returned as their last  AhjoStatus is SUBMITTED_BUT_NOT_SENT_TO_AHJO
     # and their application status is HANDLING
-    assert applications_for_open_case.count() == len(multiple_handling_applications)
+    assert applications_for_open_case.count() == len(applications_for_open_case)
```

Comment:
The two sides probably shouldn't be referring to the same variable? I think `len(applications_for_open_case)` is intended to refer to the list set on line 722 before it's overwritten on line 744.